MAUI/Button/Getting-Started.md

```diff
@@ -121,18 +126,26 @@ The button background image can be defined using the `BackgroundImageSource` pro
 {% tabs %}
 {% highlight xaml %}
 
-<buttons:SfButton x:Name="SfButton" Text="Button" BackgroundImageSource="button_background.png" CornerRadius="20" WidthRequest="100"/>
+<buttons:SfButton x:Name="SfButton" 
+                    Text="Nature"
+                    FontAttributes="Bold" 
+                    BackgroundImageSource="button_background.png" 
+                    CornerRadius="10" 
+                    WidthRequest="150"/>
 
 {% endhighlight %}
 {% highlight c# %}
 
 SfButton button = new SfButton();
-button.Text = "Button";
-button.WidthRequest = 100;
+button.Text = "Nature";
+button.FontAttributes = FontAttributes.Bold;
 button.BackgroundImageSource = "button_background.png";
-button.CornerRadius = new Thickness(20);
+button.CornerRadius = 10;
+button.WidthRequest = 150;
 
 {% endhighlight %}
 {% endtabs %}
 
 ![Button with background image](images/getting-started/ButtonWithBackgroundImage.png)
+
+You can find the complete getting started sample of .NET MAUI Button from this [link.](https://github.com/SyncfusionExamples/maui-button-samples)
```

Comment:
Find the complete getting started sample of the .NET MAUI Button from this [link.](https://github.com/SyncfusionExamples/maui-button-samples)